src/PortingAssistant.Client.Client/Reports/ReportExporter.cs

```diff
@@ -82,6 +82,8 @@ public ReportExporter(ILogger<ReportExporter> logger)
 
                     //project apis analsis result
                     string ApiAnalyzeFileName = ProjectName + "-api-analysis.json";
+                    projectAnalysResult.SourceFileAnalysisResults.Sort(
+                        (r1, r2) => string.Compare(r1.SourceFileName, r2.SourceFileName, StringComparison.OrdinalIgnoreCase));
```

Comment:
There was a performance regression identified- do we need to sort here?

Reply:
make compare between liteb and b more easy